src/commonControls/multiValueListControl/MultiValueListControl.ts

```diff
@@ -1438,6 +1501,42 @@ export class MultiValueListControl extends Control implements InteractionModelCo
         generator.ensureSlotValueIDsAreDefined(this.id, 'target', this.props.interactionModel.targets);
     }
 
+    private addBuiltInIntents(generator: ControlInteractionModelGenerator) {
+        this.addRemoveByOrderIntent(generator);
+    }
+
+    private addRemoveByOrderIntent(generator: ControlInteractionModelGenerator) {
+        generator.addIntent({
+            name: 'GeneralMultiValueListControl_RemoveByOrderIntent',
+            slots: [
+                {
+                    name: 'ordinal',
+                    type: 'AMAZON.Ordinal',
+                    multipleValues: {
+                        enabled: true,
+                    },
+                },
+                {
+                    name: 'item',
+                    type: 'multiValueListItem',
+                },
+                {
+                    name: 'action',
+                    type: 'action',
```

Comment:
These should be ` type: SharedSlotType.ACTION` similarly other common defined slotTypes